.pmtignore

```diff
@@ -1,3 +1,4 @@
 vendor/
 spec/
 tests/
+.DS_Store
```

Comment:
The 2.0.10 forge module has .DS_Store files in it.  This should prevent that happening with future releases.